src/pages/index.js

```diff
@@ -3,37 +3,48 @@ import FormValidator from "../components/FormValidator.js";
 import Popup from "../components/Popup.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
 import Styles from "./index.css";
 //import {initialCards, selectors}
-
-const initialCards = [
-  {
-    name: "Yosemite-Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-  },
-  {
-    name: "Lake-Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald-Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise-National-Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
+import Api from "../components/Api.js";
+import Utils from "../utils/utils.js";
+import { forEach, set } from "lodash";
+
+//API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "e8b97a08-d6fd-41c9-b787-2c46b2b6891e",
+    "Content-Type": "application/json",
   },
-  {
-    name: "Lago-di-Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg ",
-  },
-];
+});
+
+api
+  .getUserInfo()
+  .then((userData) => {
+    userInfo.setUserInfo(userData);
+  })
+  .catch((error) => {
+    console.error("Error getting user info:", error);
+  });
+
+api
+  .getCardList()
+  .then((res) => {
+    console.log("Here is your card list res =>", res);
+    if (Array.isArray(res)) {
+      res.forEach((cardData) => {
+        renderCard(cardData);
```

Comment:
* You haven't corrected another remark https://github.com/Agent610/se_project_aroundtheus/pull/1#discussion_r1779434736 

You have the `renderItems` method for rendering the array of cards